notes/SDLC.md

```diff
@@ -0,0 +1,54 @@
+# SDLC 
+## SLDC models: Waterfall vs Agile
+### Waterfull model  
+* Phases
+    1. Requirements
+    2. Design 
+    3. implementation
+    4. Verification
+    5. Maintenance
+* Projects progress one step at a time in a linear fashion until a project reaches maintenance
+	* "one should move to a phase only when its preceding phase is reviewed and verified" 
+* Pros 
+	* Model places significant focus on requirements and documentation
+	* Designed to catch issues as early as possible (see cost of a bug)
+	* Easy to identify milestones
+* Cons
+	* It is difficult to know all of the requirements when a project begins
+	* It is difficult to see how a project will evolve
+### Agile model  
+* "We are uncovering better ways of developing software by doing it and helping others do it."
+* Pillars of Agile
+	1. Individuals and Interactions over processes and tools
+	2. Working Software over comprehensive documentation
+	3. Customer Collaboration over contract negotiation
+	4. Responding to Change over following a plan
+* There are many different implementations of the agile manifesto 
+	* Scrum and Kanban are currently very popular
+### Scrum
+* Framework for managing work following the principals from the Agile Manfiesto
+* Scrum comes as a set of guidelines to be shaped to meet a team's specific needs
+	* Each teams practices evolve through sprint retros 
+	* Each team's scrum is unique
+* Sprints/stories/epics/spikes
```

Comment:
title this "Terminology" or something similar

Reply:
fixed